docs/pages/insights/insights-header.tsx

```diff
@@ -233,16 +239,30 @@ const PeriodSelect: FC<PeriodSelectProps> = memo(() => {
   )
 
   const onSuggestChange = useCallback(
-    (type: InsightPeriodSuggest) => {
-      const [, count, unit] = type.match(/^(\d+)([dMy])$/) ?? []
-
-      const value: [Date, Date] = [
-        dayjs()
-          .tz()
-          .subtract(parseInt(count ?? ""), unit as ManipulateType)
-          .toDate(),
-        dayjs().tz().subtract(1, "d").toDate(),
-      ]
+    (type: InsightMonthPeriodSuggest | InsightRangePeriodSuggest) => {
+      let value: [Date, Date]
+
+      if (type == "this-month") {
```

Comment:
If it was the 1st of the month, the result is the same as `"last-month"`.

This is because the insight page is one day behind.
So if it is the 1st of the month, you need to hide or disable `"this-month"`.

Reply:
Fixed.